### PR TITLE
Elidion patch 143

### PR DESCRIPTION
**Pull request by elidion**
Update for Chains of Domination PTR - Guides.xml has commented out file.

@Ludovicus-Maior  I wasn't sure if this was the right way to do this so i'll leave for you to review and approve/merge. Don't want to break anything with new zone data.

### CODE REVIEW

File: WoWPro/WoWPro_ZoneData.lua [hunk 1]
```diff
@@ -503,7 +503,7 @@ DefineZone9(0420, "Ruins of Ogudei", UIMapType.Micro, 0418, 213)
 DefineZone9(0421, "Reliquary Incursion@Ruins of Ogudei", UIMapType.Micro, 0418, 213)
 DefineZone9(0422, "Dread Wastes", UIMapType.Zone, 0424, nil, 0451, 0474, 0475)
 DefineZone9(0423, "Silvershard Mines!Dungeon", UIMapType.Dungeon, 0224, nil)
-DefineZone9(0424, "Pandaria", UIMapType.Continent, 0947, nil, 0371, 0376, 0379, 0388, 0390, 0418, 0422, 0433, 0504, 0507, 0516, 0554, 1502)
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:506]: This looks wrong 0390 and 1530 are both "Vale of Eternal Blossoms" and should both be in this list.

File: WoWPro/WoWPro_ZoneData.lua [hunk 2]
```diff
@@ -284,7 +284,7 @@ DefineZone9(0194, "Kezan", UIMapType.Zone, 0948, nil, 0195, 0196, 0197, 1010, 15
 DefineZone9(0195, "Kaja'mine 1", UIMapType.Micro, 0194, nil)
 DefineZone9(0196, "Kaja'mine 2", UIMapType.Micro, 0194, nil)
 DefineZone9(0197, "Kaja'mine 3", UIMapType.Micro, 0194, nil)
-DefineZone9(0198, "Mount Hyjal", UIMapType.Zone, 0012, nil, 0338, 0367, 0738, 0760, 1556)
+DefineZone9(0198, "Mount Hyjal", UIMapType.Zone, 0012, nil, 0338, 0367, 0738, 0760, 1556, 1958, 1959)
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:287]: Add two new dungeons to Firelands.

File: WoWPro/WoWPro_ZoneData.lua [hunk 3]
```diff
@@ -98,7 +98,7 @@ DefineZone9(0008, "Palemane Rock", UIMapType.Micro, 0007, nil)
 DefineZone9(0009, "The Venture Co. Mine", UIMapType.Micro, 0007, nil)
 DefineZone9(0010, "Northern Barrens", UIMapType.Zone, 0012, nil, 0011, 0092, 0279, 0825, 0859, 1339)
 DefineZone9(0011, "Wailing Caverns", UIMapType.Micro, 0010, nil)
-DefineZone9(0012, "Kalimdor", UIMapType.Continent, 0947, nil, 0001, 0007, 0010, 0062, 0063, 0064, 0065, 0066, 0069, 0070, 0071, 0076, 0077, 0078, 0080, 0081, 0083, 0085, 0088, 0097, 0103, 0106, 0198, 0199, 0249, 0327, 0524, 0775, 0776, 0782, 0785, 0891, 0907, 1203, 1332, 1333, 1338, 1469, 1527, 1534, 1535)
+DefineZone9(0012, "Kalimdor", UIMapType.Continent, 0947, nil, 0001, 0007, 0010, 0062, 0063, 0064, 0065, 0066, 0069, 0070, 0071, 0076, 0077, 0078, 0080, 0081, 0083, 0085, 0088, 0097, 0103, 0106, 0198, 0199, 0327, 0524, 0775, 0776, 0782, 0785, 0891, 0907, 1203, 1332, 1333, 1338, 1469, 1527, 1534, 1535)
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:101]: This dropped 0249, aka Uldum.  Not good.

File: WoWPro/WoWPro_ZoneData.lua [hunk 4]
```diff
@@ -689,7 +689,7 @@ DefineZone9(0615, "Throne of the Imperator@Highmaul!Dungeon", UIMapType.Dungeon,
 DefineZone9(0616, "Dragonspire Hall@Upper Blackrock Spire!Dungeon", UIMapType.Dungeon, 0036, 237)
 DefineZone9(0617, "The Rookery@Upper Blackrock Spire!Dungeon", UIMapType.Dungeon, 0036, 237)
 DefineZone9(0618, "Hall of Blackhand@Upper Blackrock Spire!Dungeon", UIMapType.Dungeon, 0036, 237)
-DefineZone9(0619, "Broken Isles", UIMapType.Continent, 0947, nil, 0626, 0627, 0628, 0629, 0630, 0634, 0641, 0645, 0646, 0647, 0648, 0650, 0671, 0672, 0676, 0680, 0694, 0695, 0696, 0702, 0709, 0714, 0717, 0718, 0719, 0726, 0740, 0741, 0748, 0790, 0858, 0905, 0971, 0972, 1642)
+DefineZone9(0619, "Broken Isles", UIMapType.Continent, 0947, nil, 0626, 0627, 0628, 0629, 0630, 0634, 0641, 0645, 0646, 0647, 0648, 0650, 0671, 0672, 0676, 0680, 0694, 0695, 0696, 0702, 0709, 0714, 0717, 0718, 0719, 0726, 0740, 0741, 0748, 0790, 0858, 0905, 0971, 0972, 1642, 1971)
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:692]: Added Skyhold.

File: WoWPro/WoWPro_ZoneData.lua [hunk 5]
```diff
@@ -1008,7 +1008,7 @@ DefineZone9(0941, "Lower Deck@The Vindicaar!Dungeon", UIMapType.Dungeon, 0830, 2
 DefineZone9(0942, "Stormsong Valley", UIMapType.Zone, 0876, nil, 1039, 1040, 1179, 1180, 1182, 1183, 1345, 1346, 1363)
 DefineZone9(0943, "Arathi Highlands Horde!Instance", UIMapType.Orphan, 0013, nil)
 DefineZone9(0946, "Cosmic#946", UIMapType.Cosmic, 0000, nil, 0101, 0572, 0947, 1550)
-DefineZone9(0947, "Azeroth", UIMapType.World, 0946, nil, 0012, 0013, 0113, 0378, 0407, 0424, 0619, 0824, 0875, 0876, 0938, 0939, 0948, 0981, 1156, 1157, 1355, 1528, 1580, 1581, 1582, 1590, 1591, 1592, 1593, 1594, 1595, 1596, 1597)
+DefineZone9(0947, "Azeroth", UIMapType.World, 0946, nil, 0012, 0013, 0113, 0378, 0407, 0424, 0619, 0824, 0875, 0876, 0938, 0939, 0948, 0981, 1156, 1157, 1355, 1528, 1580, 1581, 1582, 1590, 1591, 1592, 1593, 1594, 1595, 1596, 1597, 1923)
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:1011]: Added Bizzaro Pandaria.

File: WoWPro/WoWPro_ZoneData.lua [hunk 6]
```diff
@@ -1169,12 +1169,12 @@ DefineZone9(1322, "Winterspring!Kalimdor#1209", UIMapType.Zone, 1209, nil)
 DefineZone9(1323, "Thunder Bluff!Kalimdor#1209", UIMapType.Zone, 1209, nil)
 DefineZone9(1324, "Darnassus!Kalimdor#1209", UIMapType.Zone, 1209, nil)
 DefineZone9(1325, "Azuremyst Isle!Kalimdor#1209", UIMapType.Zone, 1209, nil)
-DefineZone9(1326, "The Exodar", UIMapType.Zone, 1209, nil) -- Collided {[103]=true,[1331]=true,count=3,[1326]=true}
+DefineZone9(1326, "The Exodar", UIMapType.Zone, 1209, nil) -- Collided {[103]=true,[1326]=true,[1331]=true,count=3}
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:1172]: Silly key reordering.

File: WoWPro/WoWPro_ZoneData.lua [hunk 7]
```diff
@@ -1169,12 +1169,12 @@ DefineZone9(1322, "Winterspring!Kalimdor#1209", UIMapType.Zone, 1209, nil)
...
 DefineZone9(1327, "Bloodmyst Isle!Kalimdor#1209", UIMapType.Zone, 1209, nil)
 DefineZone9(1328, "Mount Hyjal!Kalimdor#1209", UIMapType.Zone, 1209, nil)
 DefineZone9(1329, "Southern Barrens!Kalimdor#1209", UIMapType.Zone, 1209, nil)
 DefineZone9(1330, "Uldum!Kalimdor#1209", UIMapType.Zone, 1209, nil)
-DefineZone9(1331, "The Exodar", UIMapType.Zone, 1209, nil) -- Collided {[103]=true,[1331]=true,count=3,[1326]=true}
+DefineZone9(1331, "The Exodar", UIMapType.Zone, 1209, nil) -- Collided {[103]=true,[1326]=true,[1331]=true,count=3}
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:1177]: Silly key reordering

File: WoWPro/WoWPro_ZoneData.lua [hunk 8]
```diff
@@ -1249,8 +1249,8 @@ DefineZone9(1491, "The Robodrome@Mechagon!Dungeon", UIMapType.Dungeon, 0876, 399
 DefineZone9(1493, "Waste Pipes@Mechagon!Dungeon", UIMapType.Dungeon, 0876, 399)
 DefineZone9(1494, "The Under Junk@Mechagon!Dungeon", UIMapType.Dungeon, 0876, 399)
 DefineZone9(1497, "Mechagon City@Mechagon!Dungeon", UIMapType.Dungeon, 0876, 399)
-DefineZone9(1499, "", UIMapType.Micro, 1355, nil) -- Collided {[1524]=true,[1500]=true,[1499]=true,count=3}
-DefineZone9(1500, "", UIMapType.Micro, 1355, nil) -- Collided {[1524]=true,[1500]=true,[1499]=true,count=3}
+DefineZone9(1499, "Zin-Azshari", UIMapType.Micro, 1355, nil)
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:1252]: Genuine update!

File: WoWPro/WoWPro_ZoneData.lua [hunk 9]
```diff
@@ -1267,14 +1267,14 @@ DefineZone9(1520, "The Last Prison@The Eternal Palace!Dungeon", UIMapType.Dungeo
...
+DefineZone9(1524, "", UIMapType.Micro, 1355, nil) -- Collided {[1524]=true,[1829]=true,count=3,[1500]=true}
 DefineZone9(1525, "Revendreth", UIMapType.Zone, 1550, nil, 1644, 1663, 1664, 1665, 1675, 1676, 1699, 1700, 1735, 1744, 1745, 1746, 1747, 1748, 1750, 1755)
 DefineZone9(1527, "Uldum", UIMapType.Zone, 0012, nil) -- Collided {[1527]=true,[249]=true,count=2}
 DefineZone9(1528, "Nazjatar!Instance", UIMapType.Orphan, 0947, nil)
-DefineZone9(1530, "Vale of Eternal Blossoms", UIMapType.Zone, 0424, nil, 1579, 1600) -- Collided {[1530]=true,[390]=true,count=2}
+DefineZone9(1530, "Vale of Eternal Blossoms", UIMapType.Zone, 0424, nil, 1579, 1600) -- Collided {count=2,[1530]=true,[390]=true}
 DefineZone9(1531, "Crapopolis!Instance1531", UIMapType.Orphan, 0194, nil)
 DefineZone9(1532, "Crapopolis!Instance1532", UIMapType.Orphan, 0194, nil)
-DefineZone9(1533, "Bastion!The Shadowlands", UIMapType.Zone, 1550, nil, 1667, 1668, 1690, 1692, 1693, 1694, 1695, 1707, 1708, 1711, 1713, 1714)
+DefineZone9(1533, "Bastion!The Shadowlands", UIMapType.Zone, 1550, nil, 1666, 1667, 1668, 1690, 1692, 1693, 1694, 1695, 1707, 1708, 1711, 1713, 1714)
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:1277]: Added Necrotic Wake.

File: WoWPro/WoWPro_ZoneData.lua [hunk 10]
```diff
@@ -1594,4 +1594,49 @@ DefineZone9(1913, "Torghast!Instance1913", UIMapType.Orphan, 1543, nil)
...
+DefineZone9(1980, "TG114_Floor!Dungeon", UIMapType.Dungeon, 1543, nil)
+DefineZone9(1981, "TG115_Floor!Dungeon", UIMapType.Dungeon, 1543, nil)
+DefineZone9(1982, "TG116_Floor!Dungeon", UIMapType.Dungeon, 1543, nil)
+DefineZone9(1983, "TG120_Floor!Dungeon", UIMapType.Dungeon, 1543, nil)
+DefineZone9(1984, "TG113_Floor!Dungeon", UIMapType.Dungeon, 1543, nil)
+DefineZone9(1985, "TG117_Floor!Dungeon", UIMapType.Dungeon, 1543, nil)
+DefineZone9(1986, "TG118_Floor!Dungeon", UIMapType.Dungeon, 1543, nil)
+DefineZone9(1987, "TG119_Floor!Dungeon", UIMapType.Dungeon, 1543, nil)
+DefineZone9(1988, "TG121_Floor!Dungeon", UIMapType.Dungeon, 1543, nil)
+DefineZone9(1989, "The Veiled Market@Tazavesh, the Veiled Market!Dungeon", UIMapType.Dungeon, 1550, 423)
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:1627]: I'll bet money this name changes!

File: WoWPro/WoWPro_ZoneData.lua [hunk 11]
```diff
@@ -1594,4 +1594,49 @@ DefineZone9(1913, "Torghast!Instance1913", UIMapType.Orphan, 1543, nil)
 DefineZone9(1914, "Torghast!Instance1914", UIMapType.Orphan, 1543, nil)
 DefineZone9(1917, "De Other Side!Instance", UIMapType.Orphan, 1565, nil)
 DefineZone9(1920, "Torghast!Instance1920", UIMapType.Orphan, 1543, nil)
-DefineZone9(1921, "Torghast!Instance1921", UIMapType.Orphan, 1543, nil)
\ No newline at end of file
+DefineZone9(1921, "Torghast!Instance1921", UIMapType.Orphan, 1543, nil)
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_ZoneData.lua:1597]: New zones!

File: WoWPro/WoWPro_ZoneData.lua (hunk 3 above)
Comment (elidion) [reply WoWPro/WoWPro_ZoneData.lua:101]: Crap I thought I corrected that one.

**Pull request closed without merging**